README.md

```diff
@@ -298,6 +298,43 @@ class Test {};
 type TestType = typeof Test;
 ```
 
+## Nominal typing
+
+Flow treats classes as nominal types, whereas TypeScript treats them as
+structural types.
+
+### Flow
+```js
+class Foo {};
+class Bar {};
+
+const foo: Foo = new Bar();
+// Cannot assign `new Bar()` to `foo` because `Bar` [1] is incompatible with `Foo` [2].
+```
+
+### TypeScript
+```ts
+class Foo {};
+class Bar {};
+
+const foo: Foo = new Bar();
+// No errors!
+```
+
+You can work around this with tricks like the following:
+```ts
+class Foo {
+    private __nominal: void;
+};
+class Bar {
+    private __nominal: void;
```

Comment:
In TS 3.7, you should `declare private __nominal: void;` otherwise it's a field

Reply:
More over the hack used with `enum` which is a nominal in TS

Reply:
okay cool, do you have a link to more about what you're talking about with enums?

Reply:
okay, updated